Python/compile.c

```diff
@@ -2026,12 +2026,7 @@ compiler_visit_argannotation(struct compiler *c, identifier id,
 {
     if (annotation) {
         PyObject *mangled;
-        if (c->c_future->ff_features & CO_FUTURE_ANNOTATIONS) {
-            VISIT(c, annexpr, annotation)
-        }
-        else {
-            VISIT(c, expr, annotation);
-        }
+        VISIT(c, annexpr, annotation)
```

Comment:
By convention (to make it look less weird to humans) we add a redundant semicolon after `VISIT()` calls. Also below.